File: BriefAssistant/Models/RegistrationRequest.cs
using System.ComponentModel.DataAnnotations;
using System.Runtime.Serialization;

namespace BriefAssistant.Models
{
    public class RegistrationRequest
    {
        [Required]
        [EmailAddress]
        public string Email { get; set; }
        [Required]
        public string Password { get; set; }
        
        public string ConfirmPassword { get; set; }

        public UserType UserType { get; set; }
    }
}

File: BriefAssistant/ClientApp/src/app/account/login-register/RegistrationRequest.ts
import { UserType } from "../../shared/UserType";

export class RegistrationRequest {
    email: string;
    password: string;
    userType: UserType;
    confirmPassword: string;

    constructor() {
        this.email = "";
        this.password = "";
        this.confirmPassword = "";
        this.userType = UserType.User;
    }
}
